.github/actions/javascript/createOrUpdateStagingDeploy/createOrUpdateStagingDeploy.js

```diff
@@ -122,13 +122,20 @@ const run = function () {
                 'number',
             );
 
+            // Get the internalQA PR list, preserving the previous state of `isResolved`
+            const internalQAPRList = _.sortBy(
+                currentStagingDeployCashData.internalQAPRList,
+                'number',
+            );
```

Comment:
We are first getting the list of the InternalQA PRs from the previous issue body.